src/txn/txn.c

```diff
@@ -536,12 +536,46 @@ __txn_config_operation_timeout(WT_SESSION_IMPL *session, const char *cfg[], bool
     return (0);
 }
 
+/*
+ * __txn_conf_operation_timeout --
+ *     Configure a transactions operation timeout duration.
+ */
+static int
+__txn_conf_operation_timeout(WT_SESSION_IMPL *session, const WT_CONF *conf, bool start_timer)
```

Comment:
There's nothing wrong with this particular function, but I'm more asking in general - is there going to be a pattern of functions accepting both existing `char *cfg` and new `WT_CONF *conf` arguments? If so, it's probably worth looking into some way to avoid repeating too much code.

Reply:
Having two functions, one for config, one for conf, is generally the exception.  The issue here is that the original function `__txn_conf_operation_timeout` was used by: begin_transaction, commit_transaction, rollback_transaction.  If/when we convert begin_transaction and commit_transaction to the new way, we can get rid of the extra function. 

Sorry I didn't reply to this sooner!